test/models/fact_value_test.rb

```diff
@@ -67,6 +67,11 @@ def setup
     refute_empty results
   end
 
+  test 'should return search results for ILIKE operator on host' do
+    results = FactValue.search_for("host ~ abc")
+    assert_empty results
+  end
```

Comment:
Could you please add another test to make sure that `~` returns results when there are some?